configmodels/model_user_account.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Canonical Ltd.
+
+package configmodels
+
+import (
+	"golang.org/x/crypto/bcrypt"
+)
+
+const (
+	UserRole            = 0
+	AdminRole           = 1
+	UserAccountDataColl = "webconsoleData.snapshots.userAccountData"
+)
```

Comment:
We could separate the roles consts from the user account data collection as they are logically two different things